xmodule/modulestore/__init__.py

```diff
@@ -1324,16 +1324,24 @@ def create_course(self, org, course, run, user_id, fields=None, runtime=None, **
         """
         # clone a default 'about' overview block as well
         about_location = self.make_course_key(org, course, run).make_usage_key('about', 'overview')
-
         about_block = XBlock.load_class('about')
-        overview_template = about_block.get_template('overview.yaml')
+        about_block_dynamic = self.mixologist.mix(about_block)
```

Comment:
Can you explain why instantiating this block is necessary here when it wasn't before?

Reply:
Got it, we need to load the templates but blocks outside of edx-platform don't inherit the `ResourceTemplates` class directly.  So the safer way to access a working `get_template` method is to do it on an instantiated block with mixins.  This will work with built-in blocks and with the externalized one as well.

Let's add a comment with this explanation here, including a link to https://github.com/openedx/edx-platform/pull/37184